application-diagram-ui/src/main/resources/Diagram/DiagramViewSheet.xml

```diff
@@ -47,7 +47,16 @@
   #end
   {{html clean="false"}}
   ## Check if the query contains the parameter for getting the diagram from URL.
-  &lt;div class="diagram"
+
+  #set ($dispalyDiv = "show")
```

Comment:
This doesn't affect the display because it's more of a way to signal to the developers, 'Hey, the div should be displayed when viewing the HTML in view mode.' However, now that I think about it more, I believe a better approach would be to set displayDiv to an empty string as the default value